pkg/configauditreport/controller/controller.go

```diff
@@ -511,8 +511,8 @@ func (r *ResourceController) reconcileClusterConfig(kind kube.Kind) reconcile.Fu
 func clusterRbacReportItems(crar v1alpha1.ClusterRbacAssessmentReportList) func() (client.ObjectList, []client.Object) {
 	return func() (client.ObjectList, []client.Object) {
 		objlist := make([]client.Object, 0)
-		for _, item := range crar.Items {
-			objlist = append(objlist, &item)
+		for idx := range crar.Items {
```

Comment:
do we have to change the `for` to enable gosec?

Reply:
Either that, or add `nolint` statements. The specific warning is: `Implicit memory aliasing in for loop`